public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
How about updating these names to something closer to what we're doing?

Reply:
Totally forgot about the public folder... Sorry about that, updating the manifest and also making other changes.